src/modules/FtpServer.rb

```diff
@@ -958,7 +959,11 @@ def Import(settings)
       InitDaemon()
 
       result = true
-      Builtins.foreach(@UI_keys) do |key|
+
+      # This setting is not a part of the general UI but is needed
```

Comment:
The meaning of `this` is not clear in new context, I'd simply write it here, e.g. `"StartDaemon" setting is not a part of...`.